### ISSUE TITLE
leapp-cli: Use profile name and session name instead of IDs

### ISSUE THREAD

**Issue by nitrocode** [Noovolari/leapp#293]
**Is your feature request related to a problem? Please describe.**
It's a pain point to look up the ID of a profile or session in order to use `leapp session add` and `leapp session start`

Currently

```bash
✗ leapp profile create --profileName $LEAPP_PROFILE_NAME
profile created
✗ LEAPP_PROFILE_ID=$(leapp profile list -x | grep $LEAPP_PROFILE_NAME | awk '{ print $1 }' | head -1)
✗ echo $LEAPP_PROFILE_ID
7b60156a-4e35-4a0e-a2f5-94b37f35489a
✗ leapp session add \
  --providerType "aws" \
  --sessionType "awsIamRoleFederated" \
  --sessionName $LEAPP_PROFILE_ALIAS \
  --roleSessionName $LEAPP_PROFILE_ALIAS \
  --region $LEAPP_REGION \
  --roleArn $LEAPP_ROLE_ARN  \
  --idpUrl $LEAPP_IDP_URL \
  --idpArn $LEAPP_IDP_ARN \
  --profileId $LEAPP_PROFILE_ID
session added
✗ LEAPP_SESSION_ID=$(leapp session list -x --csv | grep $LEAPP_PROFILE_ALIAS | cut -d',' -f1)
✗ leapp session start --sessionId $LEAPP_SESSION_ID
```

**Describe the solution you'd like**
Use profile name and session name instead of IDs

This would simplify the above script

```bash
✗ leapp profile create --profileName $LEAPP_PROFILE_NAME
profile created
✗ leapp session add \
  --providerType "aws" \
  --sessionType "awsIamRoleFederated" \
  --sessionName $LEAPP_PROFILE_ALIAS \
  --roleSessionName $LEAPP_PROFILE_ALIAS \
  --region $LEAPP_REGION \
  --roleArn $LEAPP_ROLE_ARN  \
  --idpUrl $LEAPP_IDP_URL \
  --idpArn $LEAPP_IDP_ARN \
  --profileName $LEAPP_PROFILE_NAME
session added
✗ leapp session start --sessionName $LEAPP_PROFILE_ALIAS
```

**Describe alternatives you've considered**

**Additional context**
- Reference #259

**Comment by andreacavagna01** [comment]
Correct point. The session Alias by default cannot be unique, but it is way easier to remember the session to script.

Thanks for the suggestion, I think is something that can be made in the CLI function

**Comment by nitrocode** [comment]
@andreacavagna01 it would be nice too if this command can be run as soon as we install the leapp-cli without having to create the idp-url or profile.

I'd we provide the profile name and idp-url to `leapp session add` and the idp-url or profile name doesnt exist, it should just create it. That way it's nice and easy.

**Comment by andreacavagna01** [comment]
@urz9999 this is working for CLI or needs to be implemented?

**Comment by urz9999** [comment]
we already have the default profile, so if no profile is passed, we can use the default one, but the IdP URL must be created is something that only the user knows; there is no "default URL."

Also, because the alias is not unique, if you use the session alias to define a session, we have to implement a logic to show the user all the possible sessions to launch, and while this is usually not a problem in interactive mode, is something we must account to for scripting, and the user too. 

So, in short, it must be implemented.
